Sources/BraintreeCore/Date+MillisecondTimestamp.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension Date {
+    
+    var utcTimpestampMilliseconds: Int {
```

Comment:
small nit: 
```suggestion
    var utcTimestampMilliseconds: Int {
```

Reply:
I had noticed the same, updated here: https://github.com/braintree/braintree_ios/pull/1292/commits/e2aa0ea330939b6cb7ffec336c88fea90167854c